pypeit/images/rawimage.py

```diff
@@ -260,7 +260,7 @@ def process(self, par, bpm=bpm, flatimages=None, bias=None,
             if flatimages is None or flatimages.get_pixelflat() is None:
                 msgs.error("Flat fielding desired but not generated/provided.")
             else:
-                self.flatten(flatimages.get_pixelflat()*spec_illum, illum_flat=illum_flat, bpm=self.bpm)
+                self.flatten(flatimages.get_pixelflat()/spec_illum, illum_flat=illum_flat, bpm=self.bpm)
```

Comment:
This only impacts KCWI right, since non-IFUs don't have a spec_illum correct?

Reply:
Yes, that's correct